moto/backends.py

```diff
@@ -80,6 +80,8 @@
     "support": ("support", "support_backends"),
     "mediaconnect": ("mediaconnect", "mediaconnect_backends"),
     "mediapackage": ("mediapackage", "mediapackage_backends"),
+    "mediapackage": ("mediapackage", "mediapackage_backends"),
+    "mediapackage": ("mediapackage", "mediapackage_backends"),
 }
 
 
```

Comment:
Just remove this double here (it adds a new one every time you use script/scaffold.py

Reply:
I saw that. i'll remove it once we finish. I think it gets added everytime I run the script. So i'll just clean it up later